tutorials/efas/introduction/dcs5_copper_constant_properties_electrothermal.i

```diff
@@ -0,0 +1,1444 @@
+## Parameters for the standard DCS-5 geometry and mesh, units in meters
+
+ram_spacer_radius = 0.031
+ram_spacer_height = 0.020
+ram_spacer_overhang_radius = 0.01
+ram_spacer_overhang_height = 0.002
+
+cc_spacer_radius = 0.020
+cc_spacer_height = 0.00635
+
+sinter_spacer_radius = 0.020
+sinter_spacer_height = 0.027
+sinter_spacer_overhang_radius = 0.0135 ## is less than the 13.55 that actually exists
```

Comment:
Why is this less than the experiment?

Reply:
```
In UnstructuredMesh::stitch_meshes:
This mesh has 81 nodes on boundary `bottom_sinter_spacer_top' (14).
Other mesh has 55 nodes on boundary `' (16).
Minimum edge length on both surfaces is 0.0005.
In UnstructuredMesh::stitch_meshes:
Found 2 matching nodes.
```

Reply:
The adjustment to the geometry in the input file as compared to the actual experiment  was made to facilitate the meshing (need for nodes to align) step without forcing an unreasonably fine mesh